src/workflow/workflowOptions.ts

```diff
@@ -74,8 +75,39 @@ export const workflowOptions = {
     ],
     initial: 0,
   },
+  useSurf: {
+    type: (prev, values) => needSurfChoice(values),
+    name: "useSurf",
+    message:
+      "Would you like to use Surf, the TypeScript type safety tool maintained by Thala Labs",
```

Comment:
lets add the Surf link as part of the message https://aptos.dev/en/build/sdks/ts-sdk/type-safe-contract